### ISSUE TITLE
How to read large files?

### ISSUE THREAD

**Issue by ltxhhz** [espresso3389/flutter_content_resolver#5]
Hello, Nice Libary.
Can you add a path parameter or an input stream parameter to the resolve method to hold the content, since reading a large file directly into memory is not a good option.

**Comment by espresso3389** [comment]
Yes, I'm now working on [it](https://github.com/espresso3389/flutter_content_resolver/blob/main/lib/content_resolver.dart#L88). But it is still not working yet.

**Comment by sachinkhatripro** [comment]
Is there any update on this? Gives outofmemory error and crashes on large files.
